docs/sources/upgrade-guide/upgrade-v8.0/index.md

```diff
@@ -0,0 +1,64 @@
+---
+aliases:
+
+description: Guide for upgrading to Grafana v8.0
+keywords:
+  - grafana
+  - configuration
+  - documentation
+  - upgrade
+title: Upgrade to Grafana v8.0
+menutitle: Upgrade to v8.0
+weight: 2900
+---
+
+# Upgrade to Grafana v8.0
+
+{{< docs/shared "upgrade/upgrade-common-tasks.md" >}}
```

Comment:
I'd recommend using the updated `docs/shared` parameter syntax described in https://grafana.com/docs/writers-toolkit/writing-guide/shortcodes/#docsshared-shortcode.

The updated parameter syntax provides better feedback for user errors like referencing a file that does not exist.

You can also offset the headings using the shortcode if you need using the `leveloffset` parameter.

```suggestion
{{< docs/shared lookup="upgrade/upgrade-common-tasks.md" source="grafana" >}}
```

Same for all other instances.